integration_test/base/test_base.dart

```diff
@@ -15,15 +10,17 @@ class TestBase {
   }) {
     patrolTest(description,
         config: const PatrolTesterConfig(
-          settlePolicy: SettlePolicy.trySettle,
+          printLogs: true,
           visibleTimeout: Duration(minutes: 1),
         ),
         nativeAutomatorConfig: const NativeAutomatorConfig(
-          findTimeout: Duration(seconds: 10),
+          connectionTimeout: Duration(minutes: 1, seconds: 10),
+          findTimeout: Duration(seconds: 60),
+          keyboardBehavior: KeyboardBehavior.alternative,
```

Comment:
IMO, only login test case have an exception in timeout (because of backup process), so how about only set timeout for login test case. WDYT?